README.md

```diff
@@ -10,31 +10,34 @@ Python 3.7.2
 
 ### Instructions
 
-Before using Bridge, you'll need access to whatever media you're using to store the Ethereum wallet that owns or will own your Azimuth points. This could be an Urbit HD wallet containing a Master Ticket, a BIP39 mnemonic, a Ledger or Trezor hardware wallet, or an Ethereum private key or keystore file.
+To use Bridge:
 
-With that in hand, to use Bridge, [download a release](https://github.com/urbit/bridge/releases) and unzip it (`bridge-$version.zip`), then do one of the following:
+- [Download a release](https://github.com/urbit/bridge/releases)
+- Unzip it (`bridge-$version.zip`)
+- Open up your command line interface (Terminal on MacOS, Command Prompt on Windows)
+- Follow the instructions below to run Bridge
 
-#### Non-Ledger
+#### Running Bridge
 
 If you plan to authenticate and sign transactions with a Master Ticket, BIP39 mnemonic, Ethereum private key, or keystore file:
 
-1. Open up your command line interface (Terminal on MacOS, Command Prompt on Windows)
-2. `cd` into the `bridge-$version` directory
-3. Run this command: `python3 -m http.server 5000 --bind 127.0.0.1`
-4. Navigate to http://localhost:5000 using a web browser to access Bridge
+1. `cd` into the `bridge-$version` directory
+2. Run this command: `python3 -m http.server 5000 --bind 127.0.0.1`
+3. Navigate to http://localhost:5000 using a web browser to access Bridge
 
-#### Ledger support
+#### Running Bridge with Ledger support
 
-To authenticate and sign transactions with a [Ledger](https://www.ledger.com/), Bridge must be serving over HTTPS on localhost. This requires self-signed certificates. To do this, follow these steps:
+If you plan to authenticate and sign transactions with a [Ledger](https://www.ledger.com/), Bridge must be serving over HTTPS on localhost. This requires self-signed certificates. To do this:
 
 1. Install [mkcert](https://github.com/FiloSottile/mkcert)
 2. If you're using Firefox, additionally install [nss](https://github.com/nss-dev/nss)
 3. Install a local certificate authority via `mkcert -install`
 4. In the `bridge-$version` directory, generate a certificate valid for localhost via
   `mkcert localhost`.  This will produce two files: `localhost.pem`, the local
   certificate, and `localhost-key.pem`, its corresponding private key
-5. Run `python bridge-https.py` from the `bridge-$version` directory
-6. Navigate to https://localhost:4443 in a web browser to access Bridge
+5. `cd` into the `bridge-$version` directory
```

Comment:
But step 4 already assumes you're in that dir? This should just swap places with step 4 I think. (And then you can be less verbose in step 6 about "from this dir".)

Reply:
Ah yes – of course. Good catch.